aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutbox.java

```diff
@@ -55,15 +55,15 @@
     private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-datastore");
 
     private final LocalStorageAdapter storage;
-    private final LinkedList<PendingMutation<? extends Model>> mutationQueue;
+    private final MutationLinkedList mutationQueue;
```

Comment:
How about `private final MutationQueue mutationQueue`, as the data structure's name.

Reply:
@jamesonwilliams good call. will revise